pages/index.tsx

```diff
@@ -1,6 +1,13 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from "react";
+import Head from "next/head";
+import Image from "next/image";
+import styles from "@/styles/Home.module.css";
+import NewMembersCard from "@/src/components/new-member";
```

Comment:
Please sort imports check readme for reference

Reply:
done